docker-compose.yaml

```diff
@@ -1,12 +1,12 @@
-version: '3'
-
 services:
   arlocal:
+    profiles: ['test']
     image: textury/arlocal:v1.1.35
     ports:
       - '1984:1984'
 
   arns-service:
+    profiles: ['test', '', 'litestream']
```

Comment:
the standard here is for the default services to have no profile. but you'll need docker compose 2.x+ to support that

Reply:
nice! [c8d2714](https://github.com/ar-io/arns-service/pull/115/commits/c8d2714145a7340da0dd671a81a1dc90ade43d44)